docs/_content/api/models/common/__init__.md

```diff
@@ -0,0 +1,53 @@
+# NCO Methods Overview
+
+
+We categorize NCO approaches (which are in fact not necessarily trained with RL!) into the following: 1) constructive, 2) improvement, 3) transductive.
+
+
+```{eval-rst}
+.. tip::
+   Note that in RL4CO we distinguish the RL algorithms and the actors via the following naming:
+
+   * **Model:** Refers to the reinforcement learning algorithm encapsulated within a `LightningModule`. This module is responsible for training the policy.
+   * **Policy:** Implemented as a `nn.Module`, this neural network (often referred to as the *actor*) takes an instance and outputs a sequence of actions, :math:`\pi = \pi_0, \pi_1, \dots, \pi_N`, which constitutes the solution.
+
+   Here, :math:`\pi_i` represents the action taken at step :math:`i`, forming a sequence that leads to the optimal or near-optimal solution for the given instance.
+```
```

Comment:
We could mention here or somewhere else that abstract classes under `rl4co/models/common` are not expected to be directly initialized. For example, if you want to use an autoregressive policy, you may want to init an AM model instead of the `AutoregressivePolicy()`, same as NAR, improvement, and transductive classes.